hack/observability/cluster-api-visualizer/chart/values.yaml

```diff
@@ -1,15 +1,10 @@
-# Default values for capi-visualization.
-# This is a YAML-formatted file.
-# Declare variables to be passed into your templates.
-
-replicaCount: 1
+replicas: 1
```

Comment:
what's the difference between replicas and replicaCount in Helm syntax?

Reply:
All the variables in values.yaml are user defined. I realized that in the templates I was referencing `replicas` while I put `replicaCount` in values.yaml, which caused the field to be unused.